endoreg_db/serializers/video_segmentation.py

```
@@ -433,35 +421,18 @@ def save(self):
                             existing_segment.end_frame_number = end_frame
                             existing_segment.save()
                             updated_segments.append(existing_segment)
-                    else:
-                        # If no existing segment matches, create a new one
                         new_entries.append(
                             LabelRawVideoSegment(
```

Ruff (0.8.2)

Undefined name `LabelRawVideoSegment`

(F821)